src/hotspot/share/gc/shenandoah/shenandoahMarkBitMap.cpp

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright (c) 2020, Red Hat, Inc. and/or its affiliates.
```

Comment:
Was it copied from the `MarkBitMap`? Oracle's copyright needs to be left in place, methinks, along with Red Hat-s for Red Hat-s modifications.